resolwe/flow/executors/docker/run.py

```diff
@@ -94,11 +97,39 @@ def start(self):
 
         command_args['security'] = ' '.join(security)
 
-        # render Docker mappings in FLOW_DOCKER_MAPPINGS setting
-        mappings_template = SETTINGS.get('FLOW_DOCKER_MAPPINGS', [])
-        context = {'data_id': self.data_id}
-        mappings = [{key.format(**context): value.format(**context) for key, value in template.items()}
-                    for template in mappings_template]
+        # Setup Docker volumes.
+        def add_volume(volumes, kind, base_dir_name, volume, path=None, read_only=True):
+            """Add a new volume to the volumes list."""
```

Comment:
Add description of arguments.

Reply:
This is an internal helper, but ok, if you think that it should be added ;-)

Reply:
You can add just a quick description of arguments. There are just a lot of them, so it takes some time to understand them :)